setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name = 'sphinx_fossasia_theme',
-    version = '0.0.3',
+    version = '0.0.6',
```

Comment:
Then no need to bump the version for each PR. only bump the version when cutting a new release.